### DISCUSSION TITLE
桌面端无法连接移动端

### DISCUSSION THREAD

**Discussion by shikily** [getlantern/lantern-forum-cn#1058] [Q&A]
![5301732947175_ pic](https://github.com/user-attachments/assets/66a06c29-adf2-43b6-af44-3642d17d6794)
在“常见问题”中看到需要在“移动设备上找到唯一的 6 位数设备连接代码，可在设置中找到”，但我安卓机页面并没有“support code”的选项。
且，下一步是在“桌面设备上的“Pro 授权设备”窗口，点击“设备连接”按钮，
<img width="1327" alt="截屏2024-11-30 14 08 42" src="https://github.com/user-attachments/assets/1993d8b5-29b2-438b-91b9-7d6b9094e553">
但我桌面端（Mac）点授权设备后，出现的是“授权设备”的页面而不是“设备连接”。
虽然这样操作，安卓机能连上，但权限就转到手机上而不是电脑了。而且iphone这样操作不行。
以及本想通过“支持-报告问题“反映问题，但选择图片时自动跳转到主页的Discover功能了。
希望得到解决，感谢

**Comment by MaxMKMax** [comment]
@shikily 好的. 我会帮您提交反馈.

**Comment by MaxMKMax** [comment]
@shikily 您好. 请问您是否还遇到这个问题? 您有查看您是最新版本吗? 请您更新最新版本.

如果您还遇到同样问题的话请您帮我在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by shikily** [reply]
各设备的版本都是最新的stable版本。还是存在上述问题，且多次尝试在移动端按你所述的流程提交问题，但均提示错误，“我们遇到了技术问题。请稍后再试”。

**Reply by shikily** [reply]
通过桌面端成功发送报告。log号码为80136187

**Reply by MaxMKMax** [reply]
@shikily 非常感谢您!

**Comment by MaxMKMax** [comment]
@shikily 您好. 请问您是否还遇到这个问题? 如果依然还有这个问题的话请您反馈给我.
您有查看您是最新版本吗? 请您更新最新版本.

**Comment by MaxMKMax** [comment]
@shikily 非常感谢您! 对了, 我想问您的反馈一下. 最近的网络稳定吗? 又遇到什么问题吗? 如果有请告诉我. 我帮您反馈给技术部门优化和改进. 谢谢您支持蓝灯.
